src/generated/aniList.ts

```diff
@@ -4765,6 +4765,12 @@ export type AnimeInfoFragment = {
     medium?: string | null;
     large?: string | null;
   } | null;
+  nextAiringEpisode?: {
+    __typename?: 'MediaAiringSchedule';
+    airingAt?: number | null;
+    timeUntilAiring?: number | null;
+    episode?: number | null;
+  } | null;
```

Comment:
Instead of modifying this manually change the AnimeInfoFragment in graphql/aniList and regenerate the code.

Apart from that it looks good